plugins/cpu/pjrt_c_api_cpu_version_script.lds

```diff
@@ -0,0 +1,10 @@
+# Only symbols in the global section are available to other frameworks.
```

Comment:
can you add a comment about what this file is for?

Reply:
I think the comment here sums it up. This is a linker script, which defines which symbols are exported for use by other binaries (namely `torch_xla`). https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_chapter/ld_3.html

Reply:
I see.  How can we call this `GetPjrtApi` for example from the script in the plugins/cpu/README.md? Would be helpful to add an example.

Reply:
GetPjrtApi is only called indirectly through OpenXLA. The `PjRtCApiClient` handles all of the C-specific plumbing like this. I will add a more detailed plugin integration guide when this work is more mature.